tests/test_query.py

```diff
@@ -206,16 +196,14 @@ def test_big_and_small_full_separate_small_files(self):
 
         # if a uuid has related files or annotations then they
         # are downloaded as big files
-        assert bigs == ['big', 'small']
+        assert bigs == set(['big', 'small'])
```

Comment:
this too

Reply:
I will make it this way:
`assert set(bigs) == set(['big', 'small'])`
because there might be issue with the ordering of those items when converting from `set` to `list`

Reply:
Set to list/list to set conversion preserves order. Doing this technically allows for information to be lost. Tests the return values as they are.